lib/turn.rb

```diff
@@ -0,0 +1,27 @@
+class Turn
+    attr_reader :card, :guess, :feedback
```

Comment:
you have a method called `feedback`, so no need for this here. You do not need an attr_reader for all attributes -- you only need attr_readers for attributes you want to read when outside of the class.

Reply:
In fact, your handwritten method is scoped such that ruby will run your handwritten method first before looking at the attr_Reader method for feedback, so this is never used.